### ISSUE TITLE
Did you first make Whimsy in your free time or was it work-work?

### ISSUE THREAD

**Issue by acabunoc** [bwinton/ama#5]
And how did you get the idea?

**Comment by bwinton** [comment]
Apparently the Whimsycorn was born on [a Friday morning](https://github.com/bwinton/whimsy/commit/d8fe86c3b79001106631755e3a0e126ec45d6ae6), so, work-work-ish, I guess…  :smile:  (You know how Friday-projects are…)

The idea came out of a drunken conversation with @dolske, @shorlander, and @madhava at [Bellwoods Brewery](http://bellwoodsbrewery.com/).  I believe @dolske was lamenting Firefox’s thumbnailing service, and the many blank tiles on the newtab screen, and someone asked if we could replace them with animated gifs, so the next morning I took a shot at it.  It turned out to be really hard to figure out which tiles were missing, but replacing all of them was pretty easy, so that’s what I did!  :)

**Fun fact:** The original list of gifs was stored in [an etherpad](https://firefox-ux.etherpad.mozilla.org/thumbnail-gifs) that _anyone_ could edit.  Amazingly, no-one added anything that would get me fired!  Sadly, around 1000 gifs, etherpad broke, and stopped returning the text, so now the list of gifs [lives on github](https://github.com/bwinton/whimsy/blob/gh-pages/thumbnail-gifs.txt), where I have a little control over what gets in.
